gatsby-node.js

```diff
@@ -1,5 +1,6 @@
 const path = require(`path`)
 const { createFilePath } = require('gatsby-source-filesystem')
+const { fmImagesToRelative } = require("gatsby-remark-relative-images")
```

Comment:
Do we happen to know what `fm` means?

Reply:
[`fm` for `frontmatter`](https://github.com/danielmahon/gatsby-remark-relative-images#to-convert-frontmatter-images)